### ISSUE TITLE
Error in Golden Features: Please provide the true labels explicitly through the labels argument.

### ISSUE THREAD

**Issue by pplonski** [mljar/mljar-supervised#201]
Error when training with Golden Features:

```py
y_true contains only one label (1). Please provide the true labels explicitly through the labels argument.
```

**Comment by pplonski** [comment]
Fixed. Added advanced sampling of data with `sklearn` `train_test_split`.
